lib/biokbase/Transform/script_utils.py

```diff
@@ -136,7 +136,7 @@ def parse_docs(docstring=None):
     remainder = script_details["Args"]
     argument_values = list()
     for k in reversed(argument_keys):
-        remainder, value = remainder.split(k)        
+        remainder, _, value = remainder.rpartition(k)       
```

Comment:
I have a fix for the problem this is meant to address which I will submit a
PR for review.

On Wed, May 13, 2015 at 5:01 PM, Roman Sutormin <notifications@github.com>
wrote:

> In lib/biokbase/Transform/script_utils.py
> <https://github.com/kbase/transform/pull/151#discussion_r30287925>:
>
> > @@ -136,7 +136,7 @@ def parse_docs(docstring=None):
> >      remainder = script_details["Args"]
> >      argument_values = list()
> >      for k in reversed(argument_keys):
> > -        remainder, value = remainder.split(k)
> > +        remainder, _, value = remainder.rpartition(k)
>
> Is it possible to rename name field into something like mzml_file_name?
> It will allow us to be independent in this PR from changes in core file
> like script_utils.py which is not so easy to get merging approval for.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kbase/transform/pull/151/files#r30287925>.
>